georinex/rio.py

```diff
@@ -79,7 +79,7 @@ def first_nonblank_line(f: TextIO, max_lines: int = 10) -> str:
 
     line = ""
     for _i in range(max_lines):
-        line = f.readline(80)
+        line = f.readline(81)
```

Comment:
I don't think there's any harm in this. There could be extraneous leading space in a file. The point of this limit is in case one accidentally reads a binary or other unexpected file.

Reply:
Yes, I forgot to point it out. With 80,  it did not seem to catch the final '\n' : I thought it simply was that in version d the max length of comments go from 60 to 80, but did not investigate more than that. As you say this is just a safety net...